docs/manual/src/proc_macro/index.md

```diff
@@ -138,6 +138,29 @@ fn do_something(foo: MyFooRef) {
 }
 ```
 
+### Renaming exported items
+
+A single exported item can specify an alternate name to be used by the binding by specifying a `name` attribute.
+
+```rust
+#[uniffi::export(name=something)]
```

Comment:
```suggestion
#[uniffi::export(name = something)]
```

Reply:
Also I wonder if a bare word should be used here, or a string. If it's a bare word, using a keyword in that position will likely result in it being highlighted, but other than that I can't think of any practical consequences.

Reply:
I was going back and forward here. My initial thought was that an ident is closer to the intent and would help avoid obviously incorrect values (eg, with a space), but the more I look at it, the more I think it probably should just be a string - having an ident there makes my brain pause as I scan over it, wondering where this `something` came from. Making it a string makes it clearer we are just assigning a literal value and as you mention, causes less surprising syntax highlighting..